src/tir/schedule/concrete_schedule.cc

```diff
@@ -356,6 +356,17 @@ Array<BlockRV> ConcreteScheduleNode::GetConsumers(const BlockRV& block_rv) {
 
 /******** Schedule: Transform loops ********/
 
+LoopRV ConcreteScheduleNode::Merge(const Array<LoopRV>& loop_rvs) {
+  CHECK(!loop_rvs.empty()) << "ValueError: 'merge' requires at least 1 loop(s)";
```

Comment:
I recommend to require at least 2 loops, as "merge" 1 loop is trival

Reply:
OK